test/4.example.js

```diff
@@ -5,7 +5,7 @@
 import React, { Component } from 'react';
 
 type Props = {
-  firstName: Object,
+  firstName: number,
```

Comment:
Why was this changed from `Object`?

Reply:
Flow doesn’t treat the `Object` type the same way anymore. It acts like the `any` type now, so this change was needed so that passing a string as the `firstName` prop would continue to produce an error.

Also, Flow highly discourages using `Object` now (https://flow.org/en/docs/types/objects/#toc-object-type):

> **`Object` is unsafe and should be avoided.**